CHANGELOG.txt

```diff
@@ -113,6 +113,9 @@
         # In 'creme_core.templatetags.creme_bricks':
             - 'brick_action' accepts a new boolean argument 'comeback' ; once enabled it will add a GET argument
               'callback_url' with the current location (client side).
+        # Remove obsolete javascript tools : all browsers are supporting flexbox, table layout and ResizeObserver
+            - creme.widget.Container
+            - creme.layout.LayoutResizeSensor
```

Comment:
You are in the "non breaking change" section, but they are breaking changes.

Reply:
get lost in changelog sorry